pom.xml

```diff
@@ -139,10 +212,6 @@
 			<id>dmulloy2-repo</id>
 			<url>http://repo.dmulloy2.net/content/groups/public</url>
 		</repository>
-		<repository>
-			<id>jitpack.io</id>
-			<url>https://jitpack.io</url>
-		</repository>
```

Comment:
Why did you remove this and some more repos? Because they are hosted on the codemc repo?

Reply:
I didn't find any dependency that wasn't available in the other repos